airbyte-integrations/connectors/destination-databricks/src/test-integration/resources/sqlgenerator/alltypes_v1v2_expectedrecords_final.jsonl

```diff
@@ -1,8 +0,0 @@
-{"id1": 1, "id2": 100, "updated_at": "2023-01-01T01:00:00.000000Z", "array": ["foo"], "struct": {"foo": "bar"}, "string": "foo", "number": 42.1, "integer": 42, "boolean": true, "timestamp_with_timezone": "2023-01-23T12:34:56.000000Z", "timestamp_without_timezone": "2023-01-23T12:34:56", "time_with_timezone": "12:34:56Z", "time_without_timezone": "12:34:56", "date": "2023-01-23", "unknown": {}, "_airbyte_extracted_at": "2023-01-01T00:00:00.000000Z", "_airbyte_meta": {"changes": []}}
```

Comment:
there's no v1v2 migration test (since we never implemented the v1->v2 migration to begin with), so deleting these files